packages/permission-controller/src/Permission.ts

```diff
@@ -1,11 +1,12 @@
+import { NonEmptyArray } from '@metamask/controller-utils';
 import { Json } from '@metamask/types';
 import { nanoid } from 'nanoid';
-import { NonEmptyArray } from '@metamask/controller-utils';
+
 import { CaveatConstraint } from './Caveat';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import type { PermissionController } from './PermissionController';
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 import type { Caveat } from './Caveat';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why are we importing this type if we aren't using it?

Reply:
I don't know. 😅 I think it was imported before, so I just kept it in.

Reply:
Looks like it's because otherwise it can't resolve the types in the docs.

Reply:
Ahh... I didn't realize you had to do this but that makes sense. Good to know.